vulkan/vulkan_backbuffer.cc

```diff
@@ -52,14 +49,14 @@ bool VulkanBackbuffer::CreateSemaphores() {
   };
 
   auto semaphore_collect = [this](VkSemaphore semaphore) {
```

Comment:
See my comment in VulkanApplication regarding capturing "this" in a closure that will be invoked during destruction.